scripts/bme_testsuite_run.sh

```diff
@@ -0,0 +1,35 @@
+set -x 
+if [[ -z $1 || -z $2 ]]; then
+	echo "usage: bme_testsuite_run.sh [smoke|persistent-{clean,verify}] [DIR]"
+	exit 1
+fi
+
+if [[ $1 = "smoke" ]]; then
```

Comment:
Why are you guys configuring Tempest as part of your run script instead of doing it as part of the steps for installing the test tools? you are already installing Tempest in https://github.com/osic/qa-jenkins-onmetal/pull/27/files#diff-5d973fbc28f878292bdc7d89cb546fc1R19